src/sentry/scripts/tsdb/cmsketch.lua

```diff
@@ -373,26 +329,23 @@ function Command:new(fn, readonly)
     end
 
     return function (keys, arguments)
-        local defaults = nil
-        if not readonly then
-            defaults, arguments = (
-                function (depth, width, index, ...)
-                    return {
-                        -- TODO: Actually validate these.
-                        depth=tonumber(depth),
-                        width=tonumber(width),
-                        index=tonumber(index)
-                    }, {...}
-                end
-            )(unpack(arguments))
-        end
+        local configuration, arguments = (
+            function (depth, width, index, ...)
+                return {
+                    -- TODO: Actually validate these.
+                    depth=tonumber(depth),
+                    width=tonumber(width),
+                    index=tonumber(index)
+                }, {...}
+            end
+        )(unpack(arguments))
 
         local sketches = {}
-        for i = 1, #keys, 3 do
+        for i = 1, #keys, 2 do
```

Comment:
For some reason it took me a few mins to put this together with how this `Command:new` pattern works - I wanted there to be other code that also phase-shifts from 3 to 2, but I realized it really just corresponds to the API changes described in comments above and it makes sense.